src/frontend/apps/.cookiecutter/{{cookiecutter.app_name}}/data/queries/index.tsx

```diff
@@ -0,0 +1,125 @@
+import {
+  useMutation,
+  UseMutationOptions,
+  useQuery,
+  useQueryClient,
+  UseQueryOptions,
+} from 'react-query';
+
+import { APIList } from 'types/api';
+import { createOne } from 'data/queries/createOne';
+import { fetchList } from 'data/queries/fetchList';
+import { fetchOne } from 'data/queries/fetchOne';
+import { updateOne } from 'data/queries/updateOne';
+
+import { {{cookiecutter.model}} } from 'apps/{{cookiecutter.app_name}}/types/models';
+
+type {{cookiecutter.model_plural}}Response = APIList<{{cookiecutter.model}}>;
+type Use{{cookiecutter.model_plural}}Params = {} | { organization: string };
```

Comment:
```suggestion
type Use{{cookiecutter.model_plural}}Params = { organization: Maybe<string> };
```

I'm modifying it in this commit https://github.com/openfun/marsha/pull/1623/commits/03eed5c36f7a082a1c776007652b210ea9338d49

Reply:
And lot of modification will be merged with this commit about type and react-query fetchList

Reply:
Do you think I should wait for your commit to be merged before including those modifications ?
Because if I include them now, the generated code will be broken…